field.go

```diff
@@ -117,7 +117,12 @@ func (f *Field) Field(name string) *Field {
 // FieldOk returns the field from a nested struct. The boolean returns whether
 // the field was found (true) or not (false).
 func (f *Field) FieldOk(name string) (*Field, bool) {
-	v := strctVal(f.value.Interface())
+	value := &f.value
+	if f.value.Kind() != reflect.Ptr {
+		a := f.value.Addr()
+		value = &a
+	}
```

Comment:
Mind adding a comment why you take an address here so in the future we know what's happened :)